tests/unit/core/components/TestInstallButton.js

```diff
@@ -34,11 +34,13 @@ describe('<InstallButton />', () => {
     getClientCompatibility: () => ({ compatible: true }),
     hasAddonManager: true,
     i18n: getFakeI18nInst(),
+    store: createStore().store,
     ...customProps,
   });
 
-  const render = (props) =>
-    shallow(<InstallButtonBase {...renderProps(props)} />);
+  const render = (props) => shallowUntilTarget(
+    <InstallButton {...renderProps(props)} />, InstallButtonBase
```

Comment:
There are a couple other options:

* We could hard code the number of HOCs we wrap in the tests and call the function like `shallowUntilTarget(<Component /> { hocToUnwrap: 4 })`. But then we'd have to manually update that number any time we add/remove an HOC which I don't like.
* We could export a list of HOCs in the component and then use that in the tests like: `shallowUntilTarget(<MyComponent /> { hocToUnwrap:  MyComponentHOCs.length })`. This really isn't too different.

Considering the options, I still think looking for a base component is the best one.